app/controllers/katello/api/v2/host_errata_controller.rb

```diff
@@ -17,6 +17,16 @@ def resource_class
       Erratum
     end
 
+    def_param_group :bulk_errata_ids do
+      param :included, Hash, :required => true, :action_aware => true do
+        param :search, String, :required => false, :desc => N_("Search string for erratum to perform an action on")
+        param :ids, Array, :required => false, :desc => N_("List of errata ids to perform an action on, (ex: RHSA-2019:1168)")
+      end
+      param :excluded, Hash, :required => true, :action_aware => true do
+        param :ids, Array, :required => false, :desc => N_("List of errata ids to exclude and not run an action on, (ex: RHSA-2019:1168)")
+      end
+    end
```

Comment:
Can included and excluded be used together? might be a good thing to mention in the API docs

Reply:
Here is what I would suggest based on the behavior I am seeing:

```diff
diff --git a/app/controllers/katello/api/v2/host_errata_controller.rb b/app/controllers/katello/api/v2/host_errata_controller.rb
index 398808c..0c34460 100644
--- a/app/controllers/katello/api/v2/host_errata_controller.rb
+++ b/app/controllers/katello/api/v2/host_errata_controller.rb
@@ -18,11 +18,11 @@ module Katello
     end
 
     def_param_group :bulk_errata_ids do
-      param :included, Hash, :required => true, :action_aware => true do
+      param :included, Hash, :desc => N_("Errata to exclusively include in the action"), :required => true, :action_aware => true do
         param :search, String, :required => false, :desc => N_("Search string for erratum to perform an action on")
         param :ids, Array, :required => false, :desc => N_("List of errata ids to perform an action on, (ex: RHSA-2019:1168)")
       end
-      param :excluded, Hash, :required => true, :action_aware => true do
+      param :excluded, Hash, :desc => N_("Errata to explicitly exclude in the action. All other applicable errata will be included in the action, unless an include paramete
         param :ids, Array, :required => false, :desc => N_("List of errata ids to exclude and not run an action on, (ex: RHSA-2019:1168)")
       end
     end
```
(with better formatting of course)